angelsrefining/prototypes/override/refining-override-water-treatment.lua

```diff
@@ -176,7 +183,7 @@ if mods["bobplates"] then
     angelsmods.functions.disable_barreling_recipes("pure-water")
   end
 
-  --Insert water resources to bob recipes
+  --Insert water resources to bob recipes (NEED A WAY TO PATCH A SPECIFIC TINT)
```

Comment:
you can add recipe_tints to the patch_recipes FYI?

Reply:
I was referring to a "single" tint, but that is silly, so I will leave it as is, I am using the `crafting_machine_tint` in `patch_recipe` already, and it works quite well.